tracer/src/Datadog.Trace/AppSec/Waf/Initialization/WafConfigurator.cs

```diff
@@ -191,8 +194,9 @@ internal UpdateResult Update(IntPtr wafBuilderHandle, ConfigurationState configu
 
                 if (diags.HasErrors)
                 {
-                    // TODO: This message should go to telemetry logs only, skipping the regular logs
-                    Log.Debug("Some errors were found while applying waf configuration (RulesFile: {RulesFile})", rulesFile);
+#pragma warning disable DDLOG004 // Message templates should be constant
```

Comment:
```suggestion
               // We explicitly disable this here only because we know that the values we're sending
                // are not going to contain PII, and that we will only send the value very rarely
#pragma warning disable DDLOG004 // Message templates should be constant
```